
As upstream still have no nice fix for this, let's silence it for
now as there is nothing we can do...

https://bugs.mageia.org/show_bug.cgi?id=14869

Signed-off-by: Thomas Backlund <tmb@mageia.org>

--- linux/drivers/gpu/drm/i915/intel_fifo_underrun.c.orig
+++ linux/drivers/gpu/drm/i915/intel_fifo_underrun.c
@@ -221,7 +221,7 @@ static void cpt_set_fifo_underrun_report
 
 		if (old && I915_READ(SERR_INT) &
 		    SERR_INT_TRANS_FIFO_UNDERRUN(pch_transcoder)) {
-			DRM_ERROR("uncleared pch fifo underrun on pch transcoder %c\n",
+			DRM_DEBUG("uncleared pch fifo underrun on pch transcoder %c\n",
 				  transcoder_name(pch_transcoder));
 		}
 	}
@@ -372,6 +372,6 @@ void intel_pch_fifo_underrun_irq_handler
 {
 	if (intel_set_pch_fifo_underrun_reporting(dev_priv, pch_transcoder,
 						  false))
-		DRM_ERROR("PCH transcoder %c FIFO underrun\n",
+		DRM_DEBUG("PCH transcoder %c FIFO underrun\n",
 			  transcoder_name(pch_transcoder));
 }
